### ISSUE TITLE
Доработка авто unserialize типа text/html поля.

### ISSUE THREAD

**Issue by toxaw** [medveddev/bxorm#19]
`unserialize($this->dealer->getInfoRequisite()->getValue())['TEXT'];` приходится делать вручную. Но было бы неплохо, если было бы что-то из `$this->dealer->getInfoRequisite()->getValue()->getText();`

**Comment by medveddev** [comment]
да, пока что полноценно поддерживаются только базовые типы (скалярные, привязки к элементам и разделам), без пользовательских типов
